packages/@aws-cdk/aws-cloudformation/test/integ.core-cross-stack-string-list-references.ts

```diff
@@ -0,0 +1,70 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
```

Comment:
Was this test intended to be in the SSM package?

Reply:
nope, it's meant to be next to the other cross-stack reference tests, in `aws-cloudformation`. It happens to use string list parameters from ssm since those are simple to create.